src/test/java/no/nav/pto/veilarbportefolje/tiltakshendelse/TiltakshendelseRepositoryTest.java

```diff
@@ -90,6 +90,52 @@ public void kanLagreFlereTiltakshendelserPaSammePerson() {
         assert (tiltakshendelser.size() == 2);
     }
 
+    @Test
+    void kanLagreSletteTiltakshendelse() {
+        UUID id = UUID.randomUUID();
+        LocalDateTime opprettet = LocalDateTime.now();
+        String tekst = "Forslag: endre varighet";
+        String lenke = "http.cat/200";
+        Fnr fnr = Fnr.of("11223312345");
+
+        KafkaTiltakshendelse hendelsePaEnPerson = new KafkaTiltakshendelse(id, true, opprettet, tekst, lenke, Tiltakstype.ARBFORB, fnr, Avsender.KOMET);
+
+        assertTrue(repository.tryLagreTiltakshendelseOgSjekkOmDenErEldst(hendelsePaEnPerson));
+
+        List<Tiltakshendelse> tiltakshendelser = repository.hentAlleTiltakshendelser();
+        assert (tiltakshendelser.size() == 1);
+
+        KafkaTiltakshendelse slettHendelsePaEnPerson = new KafkaTiltakshendelse(id, false, opprettet, tekst, lenke, Tiltakstype.ARBFORB, fnr, Avsender.KOMET);
+        repository.slettTiltakshendelseOgHentEldste(slettHendelsePaEnPerson.id(), slettHendelsePaEnPerson.fnr());
```

Comment:
```suggestion
        assertNull(repository.slettTiltakshendelseOgHentEldste(id, fnr));
```

Dette forslaget har to endringar:
1: leggje til assertNull rundt slettinga så vi også sjekker at `hentEldste` returnerer null når det ikkje er fleire att i databasen.

2: Ikkje lage kafkamelding-tiltakshendelse for sletting, sidan vi berre treng `fnr` og `id`, og service-testen tester (og illustrerer) meldingsformatet på slett-meldingar.